tiledb/sm/array_schema/array_schema.h

```diff
@@ -306,11 +306,14 @@ class ArraySchema {
   /** Returns the timestamp range. */
   std::pair<uint64_t, uint64_t> timestamp_range() const;
 
+  /** Returns the the first timestamp. */
+  uint64_t timestamp() const;
```

Comment:
Maybe call this function `timestamp_start`?

Reply:
yes, changed the name to timestamp_start